From eaf468ab1640a6eda3b1f50cfb0bb53a6e7251c2 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-17.34-eaf468ab164-20211117T101025Z-avarab@gmail.com>
In-Reply-To: <cover-v4-00.34-00000000000-20211117T101025Z-avarab@gmail.com>
References: <cover-v3-00.23-00000000000-20211116T114334Z-avarab@gmail.com>
	<cover-v4-00.34-00000000000-20211117T101025Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Thu, 25 Mar 2021 12:37:00 +0100
Subject: [PATCH v4 17/34] git-repack doc: remove discussion of ancient caveat
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The backwards compatibility caveat discussed her hasn't been a concern
for anyone after Git v1.6.0. Let's simply remove it, I daresay
nobody's concerned these days with having git clients running older
than 2008-era git.

See b6945f570ac (git-repack: repo.usedeltabaseoffset, 2006-10-13) and
9f17688d93c (update git-repack documentation wrt
repack.UseDeltaBaseOffset, 2010-02-04) for the commits that previously
introduced and adjusted this documentation.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/git-repack.txt | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/Documentation/git-repack.txt b/Documentation/git-repack.txt
index 7183fb498f4..e125a591a80 100644
--- a/Documentation/git-repack.txt
+++ b/Documentation/git-repack.txt
@@ -206,16 +206,6 @@ CONFIGURATION
 Various configuration variables affect packing, see
 linkgit:git-config[1] (search for "pack" and "delta").
 
-By default, the command passes `--delta-base-offset` option to
-'git pack-objects'; this typically results in slightly smaller packs,
-but the generated packs are incompatible with versions of Git older than
-version 1.4.4. If you need to share your repository with such ancient Git
-versions, either directly or via the dumb http protocol, then you
-need to set the configuration variable `repack.UseDeltaBaseOffset` to
-"false" and repack. Access from old Git versions over the native protocol
-is unaffected by this option as the conversion is performed on the fly
-as needed in that case.
-
 Delta compression is not used on objects larger than the
 `core.bigFileThreshold` configuration variable and on files with the
 attribute `delta` set to false.
-- 
2.34.0.796.g126c2ced91c

